includes/definitions/discovery/socomec-ups.yaml

```diff
@@ -0,0 +1,275 @@
+mib: SOCOMECUPS-MIB
```

Comment:
Instead of defining mib here, it is better to just define full OIDs below.

Reply:
done ...